samples/cblas/TestBlas.java

```diff
@@ -54,18 +54,17 @@ public static void main(String[] args) {
         alpha = 1;
         beta = 0;
 
-        try (var scope = ResourceScope.newConfinedScope()) {
-            var allocator = SegmentAllocator.newNativeArena(scope);
-            var a = allocator.allocateArray(C_DOUBLE, new double[] {
+        try (var session = MemorySession.openConfined()) {
+            var a = session.allocateArray(C_DOUBLE, new double[] {
```

Comment:
Note: this could use varargs (maybe for later)

Reply:
will use varargs